apps/E2E/src/ContextualMenu/pages/ContextualMenuPageObject.win.ts

```diff
@@ -5,44 +5,31 @@ import {
   CONTEXTUALMENUITEM_TEST_COMPONENT,
 } from '../../../../fluent-tester/src/TestComponents/ContextualMenu/consts';
 import { BasePage, By } from '../../common/BasePage';
-
-/* This enum gives the spec file an EASY way to interact with SPECIFIC UI elements on the page.
- * The spec file should import this enum and use it when wanting to interact with different elements on the page. */
-export const enum ContextualMenuSelector {
-  ContextualMenu = 0, //this._primaryComponent
-  ContextualMenuItem, //this._contextualMenuItem
-}
+import { Keys } from '../../common/consts';
 
 class ContextualMenuPageObject extends BasePage {
   /******************************************************************/
   /**************** UI Element Interaction Methods ******************/
   /******************************************************************/
-  async waitForContextualMenuItemsToOpen(timeout?: number): Promise<void> {
-    await browser.waitUntil(async () => await this.contextualMenuItemDisplayed(), {
-      timeout: timeout ?? this.waitForUiEvent,
-      timeoutMsg: 'The Contextual Menu Items did not open.',
-      interval: 1000,
-    });
+  /* Waits for the MenuItems to be displayed and returns true / false whether that occurs. */
+  async waitForContextualMenuItemsToOpen(timeout?: number): Promise<boolean> {
```

Comment:
I think `WaitForContextualMenuItemsToDisplay` is  a more accurate name for this function